app/frontend/src/components/RequestForm.vue

```diff
@@ -162,6 +164,28 @@
         </v-card-text>
       </v-card>
     </v-dialog>
+
+    <!-- CHES OCIO Notice-->
+    <BaseDialog
+      v-bind:show="showChesNotice"
+      @close-dialog="showChesNotice = false"
+    >
+      <template #icon>
+        <v-icon large color="primary">check_info_outline</v-icon>
+      </template>
+      <template #text>
+        <p class="mb-2">Please read the guide on <a target="_blank"
+            href="https://github.com/bcgov/common-hosted-email-service/wiki/Best-Practices">Best Practices for use of
+            the CHES messaging API</a>.</p>
+        <p class="mb-2">There is a change to our onboarding process for CHES. Before requesting your API credentials through GETOK,
+          you must make an IStore request for a consultation with OCIO Messaging (the underlying SMTP service provider).
+          Book your consultation through your service desk (Office Productivity > Consulting Services > Messaging and
+          Collaboration - Consulting) or email <a href="mailto:mcs@gov.bc.ca">mcs@gov.bc.ca</a>.</p>
+        <p>Requests to the CHES API must originate from an IP adress within the BC Government's SpanBC network. If your
```

Comment:
'adress' misspelling

Reply:
i hate having to add this popup here. its been a while so i may have got this vuetify stuff wrong. I havent tested it locally.